arthur_bench/server/spa_static_files.py

```diff
@@ -0,0 +1,26 @@
+try:
+    import starlette
+    from starlette.staticfiles import StaticFiles
+except ImportError as e:
+    raise ImportError(
+        "Can't run Bench Server without server dependencies, to install run: "
+        "pip install arthur-bench[server]"
+    ) from e
```

Comment:
Copied from run_server, but figured there aren't any other places that we'd consider adding this, so didn't refactor this handling + error to its own file. LMK if that would be preferable.